docs/release-notes/8.19.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[release-notes-header-8.19.0]]
+== 8.19
+
+[discrete]
+[[release-notes-8.19.0]]
+=== 8.19.0
+
+[discrete]
+[[deprecations-8.19.0]]
+==== Deprecations
+* Removes default quick prompts from the Security AI Assistant ({kibana-pull}225536[#225536]).
+
+
+[discrete]
+[[features-8.19.0]]
+==== New features
+* Adds an option to update the `kibana.alert.workflow_status` field for alerts associated with attack discoveries ({kibana-pull}225029[#225029]).
+* The rule execution gaps functionality is now generally available ({kibana-pull}224657[#224657]).
+* Adds the ability to bulk fill gaps ({kibana-pull}224585[#224585]).
+* Automatic migration is now generally available ({kibana-pull}224544[#224544]).
+* Adds a name field to the automatic migration UI ({kibana-pull}223860[#223860]).
+* Adds the ability to bulk set up and delete alert suppression ({kibana-pull}223090[#223090]).
+* Adds a human-readable incremental ID to cases, making referencing cases easier ({kibana-pull}222874[#222874]).
```

Comment:
Removing since this was decided to be a no-go for 8.19/9.1.
cc: @janmonschke @michaelolo24 @paulewing 

```suggestion
```